jekyll-assets/scripts/copy-to-clipboard.js

```diff
@@ -48,7 +48,24 @@ window.addEventListener('load', function() {
       if (trigger.parentNode.querySelector('div.line')) {
         var text = extractDoxygenCode(trigger.parentNode);
       } else {
-        var text = trigger.parentNode.querySelector('pre').textContent;
+        var text = trigger.parentNode.querySelector('pre').textContent
```

Comment:
@nathan-contino removed the closing `;` ?

Reply:
We use a JavaScript formatter (e.g. [Prettier](https://prettier.io) or, more recently, [Biome](https://biomejs.dev)) to check for consistency on other projects, might it be useful here too?

Reply:
Yesssss I will add that to the list